src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionWarnOthersViewController.swift

```diff
@@ -78,6 +89,19 @@ extension ExposureSubmissionWarnOthersViewController {
 	}
 }
 
+// MARK: - Protocol ENANavigationControllerWithFooterChild
+extension ExposureSubmissionWarnOthersViewController: ENANavigationControllerWithFooterChild {
+
+	func navigationController(_ navigationController: ENANavigationControllerWithFooter, didTapPrimaryButton button: UIButton) {
+		onPrimaryButtonTap { [weak self] isLoading in
+			DispatchQueue.main.async {
+				self?.navigationFooterItem?.isPrimaryButtonLoading = isLoading
+				self?.navigationFooterItem?.isPrimaryButtonEnabled = !isLoading
+			}
+		}
+	}
```

Comment:
please keep the code inside the Mark section as it was before

Reply:
[see above](https://github.com/corona-warn-app/cwa-app-ios/pull/1609/files/246bf7f83919fb6cda893514be64ce60c5d974b7#r533956426)